QrPayment.php

```diff
@@ -94,14 +94,22 @@ public function getIBAN() {
     $part1 = ord($this->country[0]) - ord('A') + 10;
     $part2 = ord($this->country[1]) - ord('A') + 10;
 
-    $numeric = sprintf("%04d%016d%d%d00", $this->bank, $this->account, $part1, $part2);
+    $accountPrefix = 0;
+    $accountNumber = $this->account;
+    if(strpos($accountNumber, '-') !== false) {
+      $accountParts = explode('-', $accountNumber);
+      $accountPrefix = $accountParts[0];
+      $accountNumber = $accountParts[1];
+    }
+
+    $numeric = sprintf('%04d%06d%010d%d00', $this->bank, $accountPrefix, $accountNumber, $part1, $part2);
```

Comment:
Missing one %d, replace with:

`%04d%06d%010d%d%d00`

Reply:
thanks you for fix it in your repository ;-)